config/manifests/bases/elasticsearch-operator.clusterserviceversion.yaml

```diff
@@ -0,0 +1,304 @@
+      version: v1
+    - description: Kibana instance
+      displayName: Kibana
+      kind: Kibana
+      name: kibanas.logging.openshift.io
+      resources:
+      - kind: ConfigMap
+        name: ""
+        version: v1
+      - kind: ConsoleExternalLogLink
+        name: ""
+        version: v1
+      - kind: ConsoleLink
+        name: ""
+        version: v1
+      - kind: Deployment
+        name: ""
+        version: v1
+      - kind: Role
+        name: ""
+        version: v1
+      - kind: RoleBinding
+        name: ""
+        version: v1
+      - kind: Route
+        name: ""
+        version: v1
+      - kind: Service
+        name: ""
+        version: v1
+      - kind: ServiceAccount
+        name: ""
+        version: v1
+      specDescriptors:
+      - description: The node selector to use for the Kibana Visualization component
+        displayName: Kibana Node Selector
+        path: nodeSelector
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:nodeSelector
+      - description: The desired number of Kibana Pods for the Visualization component
+        displayName: Kibana Size
+        path: replicas
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podCount
+      - description: The resource requirements for the Kibana nodes
+        displayName: Kibana Resource Requirements
+        path: resources
+      statusDescriptors:
+      - description: The status for each of the Kibana pods for the Visualization component
+        displayName: Kibana Status
+        path: pods
+        x-descriptors:
+        - urn:alm:descriptor:com.tectonic.ui:podStatuses
+      version: v1
+  description: |
+    The Elasticsearch Operator for OCP provides a means for configuring and managing an Elasticsearch cluster for use in tracing and cluster logging.
+    This operator only supports OCP Cluster Logging and Jaeger.  It is tightly coupled to each and is not currently capable of
+    being used as a general purpose manager of Elasticsearch clusters running on OCP.
+
+    Once installed, the operator provides the following features:
+    * **Create/Destroy**: Deploy an Elasticsearch cluster to the same namespace in which the Elasticsearch custom resource is created.
+  displayName: Elasticsearch Operator
+  icon:
+  - base64data: ""
+    mediatype: ""
+  install:
+    spec:
+      deployments: null
+    strategy: ""
+  installModes:
+  - supported: true
+    type: OwnNamespace
+  - supported: false
+    type: SingleNamespace
+  - supported: false
+    type: MultiNamespace
+  - supported: true
+    type: AllNamespaces
+  keywords:
+  - elasticsearch
+  - jaeger
+  links:
+  - name: Elastic
+    url: https://www.elastic.co/
+  - name: Elasticsearch Operator
+    url: https://github.com/openshift/elasticsearch-operator
+  maintainers:
+  - email: aos-logging@redhat.com
+    name: Red Hat, AOS Logging
+  maturity: alpha
+  minKubeVersion: 1.18.3
+  provider:
+    name: Red Hat
+  version: 0.0.0
```

Comment:
should this be updated?

Reply:
This is the base file generated by bundle. The value will be overwritten by the version number we set in Makefile.

https://github.com/openshift/elasticsearch-operator/pull/566/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R160